local/gugcat/index.php

```diff
@@ -51,7 +51,8 @@
 $selectedmodule = null;
 $groupid = 0;
 $groups = null;
-$valid_22point_scale = true;
+$valid_22point_scale = false;
+$valid_23point_scale = false;
```

Comment:
I think you can reuse the same variable valid_22point_scale. I'm just concerned that Howard may be alarmed to read 23point scale because they do not have that.